crux-bench/time-series-bench/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM clojure:openjdk-11-lein-2.9.0
```

Comment:
let's split this up into a base image. aims:
* let's not download anything from Maven when the docker image runs
* let's minimise what we re-download from Maven when we're building the image
* let's see if we can remove the volume, means we won't depend on the local filesystem at all